pzp/qlr/mappe_base.qlr

```diff
@@ -1,203 +1,203 @@
 <!DOCTYPE qgis-layer-definition>
 <qlr>
-  <layer-tree-group expanded="1" checked="Qt::Checked" groupLayer="" name="">
+  <layer-tree-group expanded="1" name="" checked="Qt::Checked" groupLayer="">
     <customproperties>
       <Option/>
     </customproperties>
-    <layer-tree-group expanded="1" checked="Qt::Checked" groupLayer="" name="Mappe base">
+    <layer-tree-group expanded="1" name="Mappe base (Geoservizi WMS TI e CH)" checked="Qt::Unchecked" groupLayer="">
```

Comment:
@gacarrillor quite strange, I tried it out and the "Unchecked" status of the main group is not respected for "Mappe base" and "WMS". For "WFS" its respected but all qlr looks the same to me

Reply:
fixed in https://github.com/opengisch/pzp/pull/49/commits/ffee6bad9b02724d1f10a3b55d5e9fb6f219437c